lnpbp-0001.md

```diff
@@ -125,43 +125,43 @@ algorithm, that avoids all of those issues.
 
 For a given message `msg`, a set of unique public keys `P := {P1, P2, ..., Pn}`,
 `n > 0`, with some selected original public key `Po` from this set (`Po ∈ S`),
-and a protocol-specific `tag`, known to both parties, **commit procedure** runs
-as follows:
+and a protocol-specific `tag` known to both parties, the **commit procedure** 
+runs as follows:
 
-1. Reduce set `P` to `P*` by removing all duplicated public keys.
```

Comment:
By the way, a `Set` commonly only contains unique, de-duplicated values, right? What you probably mean here is a `List` `P` of public keys you want to de-duplicate into a set `P*`, correct?